pkg/experiment/local/assignment_service.go

```diff
@@ -0,0 +1,57 @@
+package local
+
+import (
+	"fmt"
+	"github.com/amplitude/analytics-go/amplitude"
+)
+
+const DayMillis = 24 * 60 * 60 * 1000
+const FlagTypeMutualExclusionGroup = "mutual-exclusion-group"
+const FlagTypeHoldoutGroup = "mutual-holdout-group"
+
+type AssignmentService struct {
```

Comment:
These should all be private

Reply:
DayMillis is also used in AssignmentFilter, FlagTypeMutualExclusionGroup and FlagTypeHoldoutGroup are both also used in local client